onyo/commands/edit.py

```diff
@@ -5,6 +5,7 @@
 from onyo import Repo, OnyoInvalidRepoError
 from pathlib import Path
 from ruamel.yaml import YAML, scanner  # pyre-ignore[21]
+from shlex import quote
```

Comment:
nit: this should be above `ruamel` and  under `pathlib`. (and `onyo` should be at the bottom)